fairlearn/_input_validation.py

```diff
@@ -42,12 +45,12 @@ def _validate_and_reformat_input(X, y=None, expect_y=True, enforce_binary_sensit
     :type y: numpy.ndarray, pandas.DataFrame, pandas.Series, or list
     :param expect_y: if True y needs to be provided, otherwise ignores the argument; default True
     :type expect_y: bool
-    :param enforce_binary_sensitive_feature: if True raise exception if there are more than two
-        distinct values in the `sensitive_features` data from `kwargs`; default False
-    :type enforce_binary_sensitive_feature: bool
     :param enforce_binary_labels: if True raise exception if there are more than two distinct
         values in the `y` data; default False
     :type enforce_binary_labels: bool
+    :param group_warn_threshold: threshold on the number of groups as defined by sensitive
+        features above which a warning is displayed; default None
```

Comment:
The comment should say what happens for None

Reply:
Removed this after feedback from Miro. The warning now comes from grid generator